go.mod

```diff
@@ -24,6 +24,8 @@ require (
 	sigs.k8s.io/gateway-api-inference-extension v1.0.0
 )
 
+replace sigs.k8s.io/gateway-api-inference-extension => github.com/RishabhSaini/gateway-api-inference-extension v0.0.0-20251107192434-0e0db125e842
```

Comment:
since we've got https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1839 in, should we use that one instead